system/testing/BaseTestCase.cfc

```diff
@@ -835,6 +843,32 @@ component extends="testbox.system.compat.framework.TestCase"  accessors="true"{
         } else {
             return getPageContext().getResponse().getResponse();
         }
+    }
+
+    /**
```

Comment:
@elpete can you remove this, since we talked about leveraging the exception bean approach.

Reply:
@elpete I want to merge this, but can you provide some cleanup please.